src/settings/MappingProfiles/MappingProfilesTransformationsModal/MappingProfilesTransformationsModal.js

```diff
@@ -31,7 +35,15 @@ import css from './MappingProfilesTransformationsModal.css';
 
 const initialSearchFormValues = {
   searchValue: '',
-  filters: { recordTypes: recordTypes.map(recordType => recordType.value) },
+  filters: {
+    recordTypes: RECORD_TYPES.map(({ value }) => value),
+    statuses: SELECTED_STATUSES.map(({ value }) => value),
+  },
+};
+
+const statusesFilterMap = {
+  [SELECTED_STATUS.SELECTED]: ({ fieldId }, selectedTransformations) => Boolean(selectedTransformations[fieldId]),
```

Comment:
```suggestion
  [SELECTED_STATUS.SELECTED]: ({ fieldId }, selectedTransformations) => !!selectedTransformations[fieldId],
```

Reply:
I use the current notation for consistency as we usually use Boolean for such cases. For example [here](https://github.com/folio-org/ui-data-export/blob/master/src/settings/MappingProfiles/MappingProfilesTransformationsModal/TransformationsField/processTransformations.js#L15). There are other places as well.